ci/scripts/execute_tests.sh

```diff
@@ -68,13 +68,15 @@ if [ -v CALL_STACK_TIMEOUT ]; then
   ssh -i ${SSHKEY_FILE} geode@${INSTANCE_IP_ADDRESS} "tmux new-session -d -s callstacks; tmux send-keys  ~/capture-call-stacks.sh\ ${PARALLEL_DUNIT}\ ${CALL_STACK_TIMEOUT} C-m"
 fi
 
+SCM_PROPS="-PsourceRevision=\"$(cd geode; git rev-parse HEAD)\" -PsourceRepository=\"${SOURCE_REPOSITORY}\""
```

Comment:
Does not need to be set on test builds since we don't ship the tested build.

Reply:
The file is still consumed in tests, though.  And on that front, we want to test exactly what we would be shipping, right?